include/avif/internal.h

```diff
@@ -238,6 +238,19 @@ typedef enum avifItemCategory
 #endif
 } avifItemCategory;
 
+// ---------------------------------------------------------------------------
```

Comment:
Nit: Should we also move this line also be into `#if defined(AVIF_ENABLE_EXPERIMENTAL_AVIR)`? Or just delets this line?